help/develop/getting-started.md

```diff
@@ -3,208 +3,216 @@
 
 # Getting Started with Building on Datagrok
 
-Welcome to [Datagrok](https://www.datagrok.ai), the next generation data platform!
-
-This is a guide to help you jump-start using Datagrok as an engineer, extending Datagrok
-with plugins and building applications on top of it. During this Getting Started,
-post your questions and suggestions at our [Community Forum](https://community.datagrok.ai/).
-We usually respond within less than a day.
-
-## Let's start
-
-*Take it fast.* If you are already somewhat knowledgeable with the platform, maybe due to
-previous demos we had for you or having watched some of our
-[youtube](https://www.youtube.com/channel/UCXPHEjOd4gyZ6m6Ji-iOBYg) videos,
-you can take a fast track:   
-> Watch the [20 minutes to an app](https://www.youtube.com/watch?v=tVwpRB8fikQ) video refresher,
-> and proceed directly to the programming [Exercises](exercises.md).  
->  
-When you have time, check
-[Daily Source of Information](#daily-sources-of-information) to know where to post questions
-and get info on the platform updates. Return to this document time to time to study
-platform _in depth_.
-
-*Take it step by step.* In case you prefer a clear start and dive into the platform
-stepwise, we recommend you to follow the below *Table of Contents*:
+Welcome to Datagrok, the next-generation data platform!
 
-1. [Daily Sources of Information](#daily-sources-of-information)
-2. [Datagrok Video Walkthrough](#datagrok-video-walkthrough)
-3. [Before you start: a Quick F.A.Q.](#faq-frequently-asked-questions)
-4. Datagrok Tutorials: [read here first](#datagrok-tutorials-and-exercises), [start directly](https://public.datagrok.ai/)
-5. Datagrok Exercises [read here first](#datagrok-tutorials-and-exercises), [start directly](exercises.md)
-6. Datagrok Architecture: [read here first](#datagrok-architecture), [start directly](admin/architecture.md)
-7. Extending and customizing Datagrok: [read here first](#extending-and-customizing-datagrok), [start directly](https://datagrok.ai/help/develop/extending-and-customizing)
-8. Building Datagrok Applications: [read here first](#datagrok-application-development), [start directly](how-to/build-an-app.md)
+This guide will help you jump-start using Datagrok as an engineer. In short, what you can do with Datagrok is extend it
```

Comment:
@AndSviat I find this sentence not representing the whole variety of what an engineer can do with the platform. The engineer can also use the platform to do data science tasks, run scripts, retrieve and explore data, and so on, namely to do everything the regular or advanced user can do. I suggest tweaking this sentence.

Reply:
Thanks for the tip, I updated the sentence.